Containerfile.rhel

```diff
@@ -0,0 +1,36 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} registry.access.redhat.com/ubi9 
+ARG INSTALL_PACKAGES_FROM=default
+ARG SAMBA_VERSION_SUFFIX=""
+ARG SAMBACC_VERSION_SUFFIX=""
+ARG SAMBA_SPECIFICS=daemon_cli_debug_output,ctdb_leader_admin_command
+
+MAINTAINER John Mulligan <jmulligan@redhat.com>
+
+LABEL org.opencontainers.image.title="Samba container"
+LABEL org.opencontainers.image.description="Samba container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
+LABEL org.opencontainers.image.url="https://github.com/samba-in-kubernetes/samba-container"
+
+
+COPY smb.conf /etc/samba/smb.conf
+COPY --chmod=755 install-packages-rhel.sh /usr/local/bin/install-packages.sh
+RUN echo "installing packages..."
+RUN --mount=type=secret,id=org-id --mount=type=secret,id=act-key /usr/local/bin/install-packages.sh \
```

Comment:
I find `act-key` a bit confusing. I'm assuming this is short for "account-key"? If so, I think we should just spell it out as `account-key` as "account" is not very long, and typically I see it abbreviated as "acct" and "act" is already a common english word.

Reply:
That makes sense and you are right, it's short for "account-key". I can make that change.

Reply:
I was wrong, it actually stands for "activation-key". I've changed it to that instead.